charts/studio/templates/ingress.yaml

```diff
@@ -0,0 +1,80 @@
+{{- if .Values.global.ingress.enabled -}}
```

Comment:
my 2 cents - better to have 2 ingresses - 1 for each service. it's less confusing and easy to reuse the values/vars needed
If you still go with this - still rename file to include resource name `ingress-studio.yaml` please

Reply:
It's all the same service to me, but it depends on how you look at it. 
In this case, I like having everything in one Ingress.
But agree with the renaming suggestion.